forge.default.yaml

```diff
@@ -52,6 +52,34 @@ agents:
       - forge_tool_net_fetch
       - forge_tool_fs_search
       - forge_tool_fs_undo
+      - sage
+      - task
+
+  - id: task
+    title: A sub-agent for Forge agent to perform tasks asynchronously
+    description: |-
+      Hands-on implementation agent that executes software development tasks through a structured 4-phase approach: task analysis, solution strategy, implementation, and quality assurance. Makes actual changes to codebases, runs shell commands, creates/modifies files, installs dependencies, and performs concrete development work. Use for building features, fixing bugs, refactoring code, or any task requiring actual modifications. Do not use for analysis-only tasks or when you want to explore options without making changes. Always validates changes through compilation and testing.
+    model: *advanced_model
+    system_prompt: |-
+      {{> forge-system-prompt-engineer-act.hbs }}
```

Comment:
Should have a separate system prompt since it doesn't have access to other agents.